### ISSUE TITLE
Provide a @all alias shorthand for notifying everyone about a reply

### ISSUE THREAD

**Issue by madduck** [loomio/loomio#8148]
In rare occasions, it makes sense to notify everyone in a reply, and instead of listing all names, it would be good to be able to use e.f. `@all` to achieve the same.

**Comment by robguthrie** [comment]
Is that everyone in the thread or the group? Would @thread and @group be suitable replacements for all? they could show up in autocomplete when @all was typed

**Comment by madduck** [comment]
Yes, I think having those would be better, as `@all` can be misleading.

**Comment by scheals** [comment]
Is this something still that has to be done? 

I came here from CodeTriage, apologies in advance if I did something against the etiquette of this repo.

**Comment by robguthrie** [comment]
@scheals yes. We could still benefit from this

**Comment by fredcarle** [comment]
Has anyone started work on this? I might give it a try if not.

**Comment by robguthrie** [comment]
Hi @fredcarle - nope, nobody has started work on this. You're welcome to give it a go. Let me know how you get on.

**Comment by Hamsterarsch** [comment]
Hi! there hasn't been much activity about this but it would be very useful for us as well. Arethere issues with the PR? Maybe I could try to take care of them as well.

**Comment by robguthrie** [comment]
I'm currently having a go at this one

**Comment by Hamsterarsch** [comment]
#11402 was merged and added the ability to mention groups by their handle. I supposed this should be closed

**Comment by robguthrie** [comment]
I need to document it but you can now type @group-handle to notify everyone in the group
